docs/tutorial/2_installation.md

```diff
@@ -207,10 +209,7 @@ Now that we have the necessary server files initialized, start the server:
 merlin server start
 ```
 
-With this command, the containerized server should now be started. Notice that two new files were added to the `merlin_server` folder:
-
-1. `merlin_server.pf`: A process file containing information regarding the Redis process
-2. `app.yaml`: A new `app.yaml` configuration file configured specifically for the containerized Redis server that we just started
+With this command, the containerized server should now be started. Notice that a new file was added to the `merlin_server` folder: `merlin_server.pf`. This is a process file containing information regarding the Redis process. Additionally, the `merlin_server/app.yaml` file is updated to add `broker` and `results_backend` sections that point to the server that was just started.
```

Comment:
no longer a list?

Reply:
Yeah `app.yaml` is now created in the `merlin server init` command. This was listing new files but since there's only one new file it doesn't make sense to put it in a list.